### ISSUE TITLE
Blank webview after init and build

### ISSUE THREAD

**Issue by ThomasSzabo** [wailsapp/wails#311]
**Description**

I  run`wails init` and create a new project using the React template. Then I `cd` into the new project folder and run `wails build -d`. After the build completes, I am simply calling the binary, which opens the app's view that shows a blank screen. A right click only shows "reload". I am on a Mac OS 10.14.5. The app works fine when I run `wails serve` and `npm run start` on the frontend... the browser shows everything perfectly.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a new wails project using `wails init`
2. Select the React template
3. Once wails is done creating the project, change directory into the project's folder
4. run `wails build -d`

**Expected behaviour**
I expect to see the HTML page in the app's view.

**Screenshots**
The log of `wails init` process
<img width="415" alt="Screen Shot 2019-12-27 at 8 49 11 PM" src="https://user-images.githubusercontent.com/11858527/71537460-9d048980-28ea-11ea-81d8-24cd35ad6dae.png">

<img width="1088" alt="Screen Shot 2019-12-27 at 8 50 55 PM" src="https://user-images.githubusercontent.com/11858527/71537472-bf96a280-28ea-11ea-8301-08dc725a2db5.png">

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v1.0.1 |
| Go Version    | go1.13 |
| Platform      | darwin |
| Arch          | amd64 |
| GO111MODULE   | on |
| GCC           | 4.2.1 |
| Npm           | 6.4.1 |
| Node          | v10.15.3 |

**Additional context**

**Comment by leaanthony** [comment]
Hmm... interesting. It must be something to do with the OSX version. I'll look at it over the next couple of days.

**Comment by leaanthony** [comment]
Do you get any output on the console? If not, please try: `hello -loglevel debug`

**Comment by macdaddyaz** [comment]
I'm seeing what could be the same issue. Fresh project with the Vue 2 template is only displaying a blank window.

There are warnings in the build output:
```
# github.com/wailsapp/wails/lib/renderer/webview
In file included from _cgo_export.c:4:
In file included from webview.go:26:
../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.h:88:3: warning: 'WebView' is deprecated: first deprecated in macOS 10.14 - No longer supported; please adopt WKWebView. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/WebKit.framework/Headers/WebView.h:119:12: note: 'WebView' has been explicitly marked deprecated here
In file included from _cgo_export.c:4:
In file included from webview.go:26:
../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.h:2036:19: warning: 'WebView' is deprecated: first deprecated in macOS 10.14 - No longer supported; please adopt WKWebView. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/WebKit.framework/Headers/WebView.h:119:12: note: 'WebView' has been explicitly marked deprecated here
# github.com/wailsapp/wails/lib/renderer/webview
In file included from ../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.go:26:
../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.h:88:3: warning: 'WebView' is deprecated: first deprecated in macOS 10.14 - No longer supported; please adopt WKWebView. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/WebKit.framework/Headers/WebView.h:119:12: note: 'WebView' has been explicitly marked deprecated here
In file included from ../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.go:26:
../../go/pkg/mod/github.com/wailsapp/wails@v1.0.1/lib/renderer/webview/webview.h:2036:19: warning: 'WebView' is deprecated: first deprecated in macOS 10.14 - No longer supported; please adopt WKWebView. [-Wdeprecated-declarations]
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/WebKit.framework/Headers/WebView.h:119:12: note: 'WebView' has been explicitly marked deprecated here
```

**Comment by sysreport** [comment]
Hi

This is not only on MacOSX. I have a Linux Mint 19 with the same behavior.
I am on a new folder without any files and will create a new Project. I make a wails init with and without the -name parameter and become a blank webview window.
As well if i type wails build -d in a blank folder. 

regards
thoralf

**Comment by sysreport** [comment]
I'm so sorry. The problem was a mistake from me. I have create and build a project named wails and its override the binary in my $G_BIN path.
I have deleted this binary and now it works for me.

**Comment by tmclane** [comment]
I can't duplicate this issue with the React UI when running Mojave 10.14.4: 

|Software|Version|
|--|--|
|Node|v12.14.1|
|npm|6.13.4|
|go|1.13.5|
|gcc|Apple 10.0.1 (clang-1001.0.46.4) - 4.2.1|
|Wails|1.0.2-pre3|

Unfortunately I cannot update OS X past this because of space issues on the box.
I would recommend trying a newer version of Wails and seeing if the issue goes away.

I just tested Wails 1.0.1 (released version) as well on the same environment and it works perfectly.  You get a window with a spinning Wails logo and a button that shows a dialog with the `basic()` message from the main.go.

**Comment by leaanthony** [comment]
Any updates @ThomasSzabo ?

**Comment by ThomasSzabo** [comment]
Sorry @leaanthony I moved on testing zserge's webview directly.

**Comment by leaanthony** [comment]
Cool. I'll close this.

**Comment by cafe4it** [comment]
same problem on macos 10.13.6

**Comment by leaanthony** [comment]
This issue is a year old. Please open a new ticket if you have an issue
